metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/PolarisConnectorTableService.java

```diff
@@ -236,6 +236,10 @@ public List<QualifiedName> listNames(
     @Override
     public void update(final ConnectorRequestContext requestContext, final TableInfo tableInfo) {
         final QualifiedName name = tableInfo.getName();
+
+        // Validate Iceberg branches/tags support for client compatibility
```

Comment:
this may be a moot point if we move this validation out of OSS, but we should add a test that checks whether the table service will actually swallow the unsupported client exception in certain cases and not propagate to the user (undesired behavior)